components/create-collective/CreateCollectiveForm.js

```diff
@@ -235,13 +257,19 @@ class CreateCollectiveForm extends React.Component {
                     >
                       {inputProps => (
                         <Field
+                          onChange={handleSlugChange}
                           as={StyledInputGroup}
                           {...inputProps}
-                          prepend="opencollective.com"
+                          prepend="opencollective.com/"
                           placeholder={placeholders.slug}
                         />
                       )}
                     </StyledInputField>
+                    {values.name.length > 0 && !this.state.isURLFieldTouched && (
+                      <P fontSize="Tiny" ml={150} fontStyle="italic" color="black.500">
```

Comment:
My bad, your initial implementation actually had the correct design. I must have looked at the wrong place.

```suggestion
                      <P fontSize="Tiny" ml={150}>
```